version.py

```diff
@@ -26,4 +26,4 @@
 #   historical commits sharing the same major version, in which case a
 #   minor version bump suffices.
 
-PROVISION_VERSION = '66.0'
+PROVISION_VERSION = '66.0'
```

Comment:
This should be bumped to `66.1`, and the newline at the end of the file should not be deleted.